### ISSUE TITLE
Access Denied While Installing Python Package due to package version

### ISSUE THREAD

**Issue by Ayush-Singhal28** [novemberfiveco/s3pypi#48]
In the project, we are using automated versioning of python project. So packages are pushed to private repo with the version as 0.1.0.post36+gec2b198.
So whenever we install package like
`pip install <package_name>==0.1.0.post36+gec2b198 --extra-index-url https://private-pip-repo.
`
Package are not installed and show access denied.

While we able to install package like
`pip install <package_name>==0.1.0.post36 --extra-index-url https://private-pip-repo
`
Basically, this (+) sign in package version creates an issue while parsing. Need to fix index.html

Because when we test with these changes. Things work properly
